docs/src/components/ComponentDoc/ComponentSourceManager/ComponentSourceManager.ts

```diff
@@ -1,8 +1,8 @@
+import { prettifyCode } from '@stardust-ui/docs-components'
```

Comment:
Temporary export, `SourceManager` will be extracted to `@stardust-ui/docs-components`, too